extras/ams_wrapper/src/api/models/classification_attributes_model.py

```diff
@@ -71,11 +71,15 @@ def postprocess_inference_output(self, inference_output: dict) -> str:
             if current_conf.confidence_threshold:
                 attributes = [attr for attr in attributes
                               if attr.confidence >= current_conf.confidence_threshold]
-            classification = SingleClassification(attributes)
+            if (output_name=="color"):
```

Comment:
To make this change applicable for wider range of models it would be better to avoid hardcoded subtype names. I suggest using the model output name as the subtype. That way the same model category could be used for other models like flower classification etc.